daemon/api.go

```diff
@@ -925,6 +907,25 @@ func appIconGet(c *Command, r *http.Request) Response {
 	return FileResponse(path)
 }
 
+// FXIME: can we kill this? all icons are always availble via the canonical
+//        /1.0/icons/$name.$origin/icon
+func metaIconGet(c *Command, r *http.Request) Response {
```

Comment:
So I can just remove it? No concerns about backward compatiblity? Or can we remove it on the next API change?